docs/en/enterprise-edition/content-collections/application-security/get-started/connect-code-and-build-providers/code-repositories/add-bitbucket.adoc

```diff
@@ -72,6 +72,9 @@ NOTE: The selected repositories associated with the app password will be scanned
 . Verify integration.
 .. In *Application Security*, select *Home* > *Settings* > *Code & Build Providers*.
 .. Confirm that the status of the integrated *Bitbucket* repositories displays *Connected* under *Status*.
+.. After successfully completing integration, a new webhook is displayed in Bitbucket under *Settings* > *Workflow* > *Webhooks*.
++
+image::bitbucket-6.png[width=800]
```

Comment:
Let me know the proper way to reference this image

Reply:
Hi @tsmithv11 Fixed the sytax error.

Reply:
Thanks!